gui/widget.cpp

```diff
@@ -111,6 +115,16 @@ void Widget::draw() {
 		_x = getAbsX();
 		_y = getAbsY();
 
+		if (g_gui.useRTL()) {
```

Comment:
You also need to check against `nortl` flag for the current widget

Reply:
The drawing function is here for flipping the GUI.. the `noRTL` flag will only disable an internal widget from not flipping around. They have been added to each individual widgets where necessary.